packages/cli/config/tsconfig.json

```diff
@@ -10,6 +10,7 @@
     "module": "ESNext",
     "resolveJsonModule": true,
     "esModuleInterop": true,
+    "lib": ["DOM", "DOM.Iterable", "ScriptHost", "ES2019"],
```

Comment:
Default libs for ES6 are `["DOM", "DOM.Iterable", "ScriptHost", "ES2019", "ES6"]`, and those won't be included if we specify libs here.